OWNERS

```diff
@@ -1,10 +1,12 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-  - andyzhangx
-  - ddebroy
-  - PatrickLang
-  - msau42
-  - yujuhong
-  - jingxu97
-  - kkmsft
+- kubernetes-csi-approvers
+- andyzhangx
+- ddebroy
+- PatrickLang
+- jingxu97
+- kkmsft
```

Comment:
yujuhong is missing from approvers.

Reply:
Fixed.

Reply:
@yujuhong doesn't seem to be a member of the kubernetes-csi org. How did that work before?

Reply:
hmm, maybe this owners file check was added later.

Reply:
@yujuhong can you apply to be a member of the kubernetes-csi org?  I see that you are already a member of kubernetes org.

Reply:
I've removed @yujuhong from OWNERS. As it looks like there was no past activity either (because that can't have worked), I've not added to "emeritus".

Once the org change is done, we can edit this file in another PR. But in the meantime I'd like to conclude the OWNERS changes and csi-release-tools update.

@xing-yang can you take another look?

Reply:
Sure